src/components/AppNavigation/CalendarList/Trashbin.vue

```diff
@@ -115,6 +118,11 @@ export default {
 
 			return formattedCalendars.concat(formattedCalendarObjects)
 		},
+		retentionDuration() {
+			return Math.ceil(
+				this.trashBin._props['{http://nextcloud.com/ns}trash-bin-retention-duration'] / (60 * 60 * 24)
```

Comment:
On the server side, the `calendarRetentionObligation` app config is saved in seconds, so we need to expect this value not to be an exact number of days...and I've noticed the `Math.ceil()` above afterwards, good catch. :see_no_evil:

Reply:
The check warnings are sometimes hiding things.

Reply:
Yep it's not ideal. Moment.js can't format a time duration right now, but possibly in the future we can try to translate the seconds into something human readable and then also durations like *Two and a half days* could show accordingly instead of the rounded up *3 days*.